packages/compass-web/polyfills/net/index.ts

```diff
@@ -156,8 +160,6 @@ export { isIPv4, isIPv6 } from 'is-ip';
 export const isIP = (input: string) => ipVersion(input) ?? 0;
 export const createConnection = (options: { host: string; port: number }) => {
   const socket = new Socket();
-  setTimeout(() => {
-    socket.connect(options);
-  });
+  socket.connect(options);
```

Comment:
This was making an exception uncatchable by the try/catch wrapping connect(). The exception was momentary while I was debugging stuff so not a code path used in practice, but curious to know why we put this in a setTimeout?